packages/admin/admin/src/form/Autocomplete.tsx

```diff
@@ -43,14 +42,18 @@ export const FinalFormAutocomplete = <
     loading = false,
     loadingError,
     isAsync = false,
-    clearable,
+    required,
+    disabled,
+    readOnly,
     loadingText = <FormattedMessage id="common.loading" defaultMessage="Loading ..." />,
     popupIcon = <ChevronDown />,
     noOptionsText = <FormattedMessage id="finalFormAutocomplete.noOptions" defaultMessage="No options." />,
     errorText = <FormattedMessage id="finalFormSelect.error" defaultMessage="Error loading options." />,
     ...rest
 }: FinalFormAutocompleteProps<T, Multiple, DisableClearable, FreeSolo> & FinalFormAutocompleteInternalProps<T>) => {
     const value = multiple ? (Array.isArray(incomingValue) ? incomingValue : []) : incomingValue;
+    const clearable = value !== null && !required && !disabled && !readOnly;
```

Comment:
Checking for value isn't necessary here:

```suggestion
    const clearable = !required && !disabled && !readOnly;
```